[2024-08-07T11:11:18.645+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-08-07T11:11:18.673+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_2024-08-07.decrypt_the_data scheduled__2024-07-14T00:00:00+00:00 [queued]>
[2024-08-07T11:11:18.682+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_2024-08-07.decrypt_the_data scheduled__2024-07-14T00:00:00+00:00 [queued]>
[2024-08-07T11:11:18.682+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-08-07T11:11:18.722+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): decrypt_the_data> on 2024-07-14 00:00:00+00:00
[2024-08-07T11:11:18.728+0000] {standard_task_runner.py:63} INFO - Started process 4568 to run task
[2024-08-07T11:11:18.732+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'etl_2024-08-07', 'decrypt_the_data', 'scheduled__2024-07-14T00:00:00+00:00', '--job-id', '822', '--raw', '--subdir', 'DAGS_FOLDER/final_etl.py', '--cfg-path', '/tmp/tmppywr4ejl']
[2024-08-07T11:11:18.735+0000] {standard_task_runner.py:91} INFO - Job 822: Subtask decrypt_the_data
[2024-08-07T11:11:18.811+0000] {task_command.py:426} INFO - Running <TaskInstance: etl_2024-08-07.decrypt_the_data scheduled__2024-07-14T00:00:00+00:00 [running]> on host 8cffaad11d99
[2024-08-07T11:11:19.812+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_2024-08-07' AIRFLOW_CTX_TASK_ID='decrypt_the_data' AIRFLOW_CTX_EXECUTION_DATE='2024-07-14T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-07-14T00:00:00+00:00'
[2024-08-07T11:11:19.813+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-08-07T11:11:19.835+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-08-07T11:11:19.836+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'python3 /opt/***/data/decryption_utils.py']
[2024-08-07T11:11:19.842+0000] {subprocess.py:86} INFO - Output:
[2024-08-07T11:11:19.932+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-08-07T11:11:19.933+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-08-07T11:11:19.985+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=etl_2024-08-07, task_id=decrypt_the_data, run_id=scheduled__2024-07-14T00:00:00+00:00, execution_date=20240714T000000, start_date=20240807T111118, end_date=20240807T111119
[2024-08-07T11:11:20.070+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-08-07T11:11:20.098+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-08-07T11:11:20.121+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
